core/src/main/resources/reference.conf

```diff
@@ -1400,18 +1412,29 @@ datastax-java-driver {
     # By default, this thread is named after the session name and "-timer-0", for example
     # "s0-timer-0".
     timer {
-      # The timer tick duration
-      # This is the how frequent the timer should wake-up to check for timed out tasks.
-      # Lower resolution (i.e. longer durations) will leave more CPU cycles for running I/O
-      # operations at the cost of precision of exactly when a request timeout will expire or a
+      # The timer tick duration.
+      # This is how frequent the timer should wake up to check for timed-out tasks or speculative
+      # executions. Lower resolution (i.e. longer durations) will leave more CPU cycles for running
+      # I/O operations at the cost of precision of exactly when a request timeout will expire or a
       # speculative execution will run. Higher resolution (i.e. shorter durations) will result in
       # more precise request timeouts and speculative execution scheduling, but at the cost of CPU
       # cycles taken from I/O operations, which could lead to lower overall I/O throughput.
       #
+      # The default value is 100 milliseconds, which is a comfortable value for most use cases.
+      # However if your are using more agressive timeouts or speculative execution delays, then you
+      # should lower the timer tick duration as well, so that its value is always equal to or lesser
+      # than the timeout duration and/or speculative execution delay you intend to use.
+      #
+      # Note for Windows users: avoid setting this to aggressive values, that is, anything under 100
+      # milliseconds; doing so is known to cause extreme CPU usage. Also, the tick duration must be
+      # a multiple of 10 under Windows; if that is not the case, the driver will automatically round
```

Comment:
technically netty is what is rounding things down, right?

Reply:
Is it Netty or is it the JVM implementation on Windows that can't handle a granularity smaller than 10ms?

Reply:
it's a windows thing, explained here: https://www.javamex.com/tutorials/threads/sleep_issues.shtml 

netty rounds down to nearest 10 here: https://github.com/netty/netty/blob/netty-4.1.28.Final/common/src/main/java/io/netty/util/HashedWheelTimer.java#L557-L564

Reply:
It's Netty that rounds down – do you think we should mention Netty here?

Reply:
I like your change, which is to mention that it is being rounded down (but not what is rounding it down).